### ISSUE TITLE
Flatpak/Publish on Flathub

### ISSUE THREAD

**Issue by Newbytee** [MrBn100ful/HermesMessenger#2]
It would be nice to have this on Flathub so it can be installed easily on various distributions. Have you considered this?

**Comment by MrBn100ful** [comment]
Yes i will publish the app on flathub, but firt i need to have a proper version to pulish !
